connectd/usr/bin/connectd_options

```diff
@@ -7,7 +7,7 @@
 #  Any script which includes this file must have #!/bin/sh as its first line.
 #  connectd_options is never executed directly.
 #===================================
-MODIFIED="December 02, 2019"
+MODIFIED="June 07, 20209"
```

Comment:
typo in date

Reply:
I fixed the typo, will approve and merge.